### ISSUE TITLE
[Rebuild] nvidia-zen

### ISSUE THREAD

**Issue by CerulanLumina** [chaotic-aur/packages#2946]
### Packages:

`nvidia-zen`

### Latest chaotic version

545.29.02.2+6.6.1.zen1.1-1

### Reason for rebuild

Latest `linux-zen` version is `6.6.4.zen1-1`, which is incompatible with latest `nvidia-zen` version.

### Logs

https://builds.garudalinux.org/repos/chaotic-aur/logs/nvidia-zen.log

**Comment by xiota** [comment]
@dr460nf1r3 Related to some reorganization a while ago?

404: https://gitlab.com/garuda-linux/packages/pkgbuilds/nvidia-zen

**Comment by dr460nf1r3** [comment]
> @dr460nf1r3 Related to some reorganization a while ago?
> 
> 404: https://gitlab.com/garuda-linux/packages/pkgbuilds/nvidia-zen

Yes indeed! Might be best to create a new repo for that one here.
